conf/modules.config

```diff
@@ -469,16 +469,9 @@ process {
         ext.args = { "${meta.db_params}" }
         ext.prefix = params.perform_runmerging ? { "${meta.id}_${meta.db_name}.bracken" } : { "${meta.id}_${meta.run_accession}_${meta.db_name}.bracken" }
         publishDir = [
-	    [
             path: { "${params.outdir}/bracken/${meta.db_name}/" },
             mode: params.publish_dir_mode,
-            pattern: '*.tsv'
-            ],
-	    [
-            path: { "${params.outdir}/bracken/${meta.db_name}/" },
-            mode: params.publish_dir_mode,
-            pattern: '*bracken.kraken2.report_bracken*.txt'
-	    ]
+            pattern: '*{.tsv,.report_bracken_species.txt}'
```

Comment:
Is the `.report_bracken_species` infix really necessary? What if the pattern is only `*.{txt,tsv}`?

Reply:
It is necessary, as the report from Kraken2 (the uncorrected one) also has the `.txt` suffix

Reply:
Nextflow already knows not to pick up input files :), so actually `txt,tsv` shoudl be sufficient

Reply:
Ah that is great! Learn something new every day!

Reply:
Maybe something like?

`pattern: '*{.tsv, txt}'`